src/Vehicle/Vehicle.cc

```diff
@@ -1838,6 +1838,16 @@ void Vehicle::pauseVehicle(void)
     _firmwarePlugin->pauseVehicle(this);
 }
 
+void Vehicle::abortLanding(void)
+{
+    sendMavCommand(defaultComponentId(),
+                   MAV_CMD_DO_GO_AROUND,
+                   true,        // show error if fails
+                   50);         // this is a dummy value that is currently ignored
+
+    _firmwarePlugin->abortLanding(this);
```

Comment:
Not sure why any of this needs to be in FirmwarePlugin. This is a generic mavlink command which works both on PX4 and ArduPilot stacks. So there shouldn't be anything that is vehicle specific.